### ISSUE TITLE
Adding "How to use"  and files structure In Readme.md

### ISSUE THREAD

**Issue by Purvanshsingh** [akshitagupta15june/lane_detection_opencv#65]
**Is your feature request related to a problem? Please describe.**
There are lots of files present in our repo, we add a section that will be explaining the major files on which we have to work on.

**Describe the solution you'd like**
Add a section Usage which will contain the folder structure in this way :
(sample)
![image](https://user-images.githubusercontent.com/49719371/103221502-d67e3380-4948-11eb-989b-4d1ab68078f4.png)

It will clear for all the contributor where to start from and what are the major files to be worked on.

**Describe alternatives you've considered**

**Approach to be followed (optional)**

**Additional context**

**Comment by Purvanshsingh** [comment]
I will work on this issue.

**Comment by Purvanshsingh** [comment]
@akshitagupta15june please share your views.

**Comment by akshitagupta15june** [comment]
Yes this is good!
You can work on this.

**Comment by Aayush-hub** [comment]
@akshitagupta15june Mam can you please assign me this, if the issue is still not solved! #SWOC

**Comment by Purvanshsingh** [comment]
I am working on it.

**Comment by Aayush-hub** [comment]
@akshitagupta15june Mam, can I work on this issue if its still open, as no work is being done for 23 days. #SWOC

**Comment by akshitagupta15june** [comment]
yes @Aayush-hub but i think @Purvanshsingh is still working in this.

**Comment by akshitagupta15june** [comment]
@Aayush-hub would you like to work in this issue https://github.com/akshitagupta15june/lane_detection_opencv/issues/44
it is still not assigned to anyone.

**Comment by Aayush-hub** [comment]
@akshitagupta15june Yes sure, I can give it a try:)

**Comment by Aayush-hub** [comment]
@akshitagupta15june Can I Work on this issue, as it is not solved yet?

**Comment by akshitagupta15june** [comment]
Yes you can work in it @Aayush-hub

**Comment by Aayush-hub** [comment]
@akshitagupta15june Mam, can you please review changes in Readme [here](https://github.com/Aayush-hub/lane_detection_opencv.git) and suggest any further changes required?

**Comment by akshitagupta15june** [comment]
Yes i have checked the changes.
You can create the pr
